### PR TITLE
feat: support zip command  #571

### PR DESCRIPTION
**Pull request by Leizhenpeng**

### CODE REVIEW

File: packages/zipper/index.ts [hunk 1]
```diff
@@ -0,0 +1,10 @@
+import { resolve } from 'path';
+
+import { zipBundle } from './lib/zip-bundle';
+
+// 打包根目录dist文件
```
Comment (PatrykKuniczak) [inline packages/zipper/index.ts:5]: Create description only in english, for better understanding by everybody :)

File: packages/zipper/lib/zip-bundle/index.ts [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+      onlyFiles: true,
+    },
+  );
+
+  return new Promise<void>((pResolve, pReject) => {
+    let size = 0;
+    let aborted = false;
+    const timer = Date.now();
+    const zip = new Zip((err, data, final) => {
+      if (aborted) return;
```
Comment (PatrykKuniczak) [inline packages/zipper/lib/zip-bundle/index.ts:71]: it's not better to create
```suggestion
      if (!aborted) {rest code ...}
```

Also below you have the same case

File: packages/zipper/lib/zip-bundle/index.ts [hunk 3]
```diff
@@ -0,0 +1,104 @@
...
+
+      if (err) {
+        pReject(err);
+      } else {
+        size += data.length;
+        output.write(data);
+        if (final) {
+          logPackageSize(size, timer);
```
Comment (PatrykKuniczak) [inline packages/zipper/lib/zip-bundle/index.ts:79]: Why you don't use data.length directly, this var 'size' seems have no sense, or i don't saw it 😆

File: packages/zipper/tsconfig.json [hunk 4]
```diff
@@ -0,0 +1,8 @@
+{
+  "extends": "@typetrans/tsconfig/utils",
+  "compilerOptions": {
+    "outDir": "dist",
+    "types": ["chrome","node"]
```
Comment (PatrykKuniczak) [inline packages/zipper/tsconfig.json:5]:
```suggestion
    "types": ["chrome", "node"]
```

**Review by Jonghakseo — CHANGES_REQUESTED** [review]
스크린샷 2024-07-18 오후 9 09 31

It makes a zip file with the entire nested folder structure. 🤔

**Comment by Leizhenpeng** [comment]
https://github.com/user-attachments/assets/aea84e72-9f49-4028-bcbd-550e5408a60d

@Jonghakseo I haven't reproduced the bug you said.

**Comment by Leizhenpeng** [comment]
@Jonghakseo It was my negligence. Ha ha, it has been corrected now.

**Comment by Leizhenpeng** [comment]
CleanShot 2024-07-20 at 20 46 33@2x

**Review by Jonghakseo — APPROVED** [review]
LGTM

File: packages/zipper/lib/zip-bundle/index.ts (hunk 2 above)
Comment (PatrykKuniczak) [reply packages/zipper/lib/zip-bundle/index.ts:77]: @Leizhenpeng Are you eager and able to implement this?

**Pull request merged**